lib/os/onoff.c

```diff
@@ -1,39 +1,39 @@
 /*
  * Copyright (c) 2019 Peter Bigot Consulting, LLC
+ * Copyright (c) 2020 Nordic Semiconductor ASA
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 #include <kernel.h>
 #include <sys/onoff.h>
 
-
-#define SERVICE_CONFIG_FLAGS	     \
-	(ONOFF_SERVICE_START_SLEEPS  \
-	 | ONOFF_SERVICE_STOP_SLEEPS \
-	 | ONOFF_SERVICE_RESET_SLEEPS)
+#define SERVICE_CONFIG_FLAGS \
+	(ONOFF_START_SLEEPS  \
+	 | ONOFF_STOP_SLEEPS \
+	 | ONOFF_RESET_SLEEPS)
 
 #define SERVICE_REFS_MAX UINT16_MAX
 
-#define SERVICE_STATE_OFF 0
-#define SERVICE_STATE_ON ONOFF_SERVICE_INTERNAL_BASE
-#define SERVICE_STATE_TRANSITION (ONOFF_SERVICE_INTERNAL_BASE << 1)
-#define SERVICE_STATE_TO_ON (SERVICE_STATE_TRANSITION | SERVICE_STATE_ON)
-#define SERVICE_STATE_TO_OFF (SERVICE_STATE_TRANSITION | SERVICE_STATE_OFF)
+#define ST_OFF 0
```

Comment:
about naming, I don't know if going to the shortest possible one is relevant (it becomes cryptic then). You'll need to find something in the middle.